src/test/java/com/moodybluez/enterprise/EnterpriseApplicationTests.java

```diff
@@ -81,9 +81,9 @@ private void whenEntryIsCompleted() {
 		entry.setReasonDesc(reasonForMood);
 		entry.setDate(new Date());
 		entry.date.setDate(entryDate);
-		entry.date.setDayOfWeekID(dayOfWeek.getValue());
-		entry.date.setDayOfWeekDesc(dayOfWeek.name());
-		entry.date.setDayOfWeekID(dayOfWeek.getValue());
+		entry.date.setDayID(dayOfWeek.getValue());
+		entry.date.setDayDesc(dayOfWeek.name());
+		entry.date.setDayID(dayOfWeek.getValue());
```

Comment:
I changed variables in the classes, so I had to change them in the unit test also.